pkg/command/command.go

```diff
@@ -129,7 +129,9 @@ func (c *Command) RunShell(shell, script string) error {
 func (c *Command) RunExecutable(executable string, params ...string) error {
 	c.prepareOut()
 
-	cmd := ExecCommand(executable, params...)
+	rawCmd := strings.Join(params, " ")
+	finalParams := os.ExpandEnv(rawCmd)
+	cmd := ExecCommand(executable, finalParams)
```

Comment:
To make it work, we need to pass the finalParams as a slice of strings, otherwise there will be an error. 
For the helm lint command there will be error: unknown command "lint ../test-helm-chart" for "helm".